lib/helpers/copy-fixture-files.js

```diff
@@ -2,13 +2,11 @@
 
 var path    = require('path');
 var Promise = require('rsvp');
-var copy    = Promise.denodeify(require('cpr'));
+var fs = require('fs-extra');
+var copy = Promise.denodeify(fs.copy);
 
 var rootPath = process.cwd();
 
 module.exports = function copyFixtureFiles(sourceDir) {
-  return copy(path.join(rootPath, 'tests', 'fixtures', sourceDir), '.', {
-    clobber: true,
-    stopOnErr: true
-  });
+  return copy(path.join(rootPath, 'tests', 'fixtures', sourceDir), '.', { clobber: true });
```

Comment:
the problem is that the most recent release of `cpr` doesn't have either of those options anymore (see https://github.com/davglass/cpr#options) so I thought that we might as well use `fs-extra` then.